util/promlint/promlint.go

```diff
@@ -205,6 +210,54 @@ func lintHistogramSummaryReserved(mf dto.MetricFamily) []Problem {
 	return problems
 }
 
+// lintMetricTypeInName detects when the type of the metric is included in the metric name
```

Comment:
Missing dot at the end of the sentence and the same for the other comments.

Reply:
Fixed.